Sources/SpeziLLMLocal/LLMLocalSession+Generate.swift

```diff
@@ -9,18 +9,18 @@
 import Foundation
 import MLX
 import MLXLLM
+import MLXLMCommon
 import MLXRandom
 import os
 import SpeziChat
 import SpeziLLM
 
 
 extension LLMLocalSession {
-    // swiftlint:disable:next identifier_name function_body_length
+    // swiftlint:disable:next identifier_name function_body_length cyclomatic_complexity
     internal func _generate(continuation: AsyncThrowingStream<String, any Error>.Continuation) async {
 #if targetEnvironment(simulator)
-        // swiftlint:disable:next return_value_from_void_function
-        return await _mockGenerate(continuation: continuation)
+        return await _mockGenerate(continuation: continuation) // swiftlint:disable:this return_value_from_void_function
```

Comment:
We have a lot of SwiftLint disable calls in this file and others that are not needed and can be resolved by following the guidelines for the lint rules, e.g.:
```suggestion
       await _mockGenerate(continuation: continuation)
```